src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala

```diff
@@ -140,7 +141,7 @@ class ReviewConsensusVariants
     !rec.getReadUnmappedFlag && variantsByChromAndPos(rec.getContig).exists { v =>
       if (v.start >= rec.getAlignmentStart && v.start <= rec.getAlignmentEnd) {
         val readPos = rec.getReadPositionAtReferencePosition(v.start)
-        readPos == 0 || !SequenceUtil.basesEqual(rec.getReadBases()(readPos-1), v.refBase.toByte)
+        readPos == 0 || !SequenceUtil.basesEqual(rec.getReadBases()(readPos-1), v.refBase.toByte) && (!this.ignoreN || !SequenceUtil.isNoCall(rec.getReadBases()(readPos-1)))
```

Comment:
Probably should extract `val base = rec.getReadBases()(readPos-1)` since it's being used twice now.